### ISSUE TITLE
Uninstall XeroSploit

### ISSUE THREAD

**Issue by lazycipher** [LionSec/xerosploit#145]
Is there any way to uninstall XeroSploit ?
I found no solution to this on internet.

**Comment by sarthak1598** [comment]
Yes , there is a well formulated way in the UNIX/LINUX to uninstall any installed packages/software
# use this commands 
# sudo apt remove xerosploit  or if not works !!! 
#use this one 

# sudo apt purge xerosploit

**Comment by azamet90** [comment]
> Yes , there is a well formulated way in the UNIX/LINUX to uninstall any installed packages/software
> # use this commands
> # sudo apt remove xerosploit or if not works !!!
> 
> #use this one
> # sudo apt purge xerosploit

did this work on you or you just doing so like you help? because of we did not install the package from kali repo ?
